src/containers/catalog/CatalogComponent.js

```diff
@@ -0,0 +1,21 @@
+import React, { PropTypes } from 'react';
+import { EntitySetList } from './components/EntitySetList';
+import AuthService from '../../utils/AuthService';
+
+
+export class CatalogComponent extends React.Component {
+  static propTypes = {
```

Comment:
I've been following this style...

```
class Organizations extends React.Component {
  ...
}

Organizations.propTypes = {
  ...
};
```

... but I don't really have a solid case for why, just what I've seen in tutorials. Any ideas as to which approach is preferred, using `static` inside the class, or setting it on the class outside?